### PR TITLE
IN-141: Added standard labels to all yaml files.

### PR DESCRIPTION
**Pull request by michaelamiethyst**

### CODE REVIEW

File: charts/kibana/templates/kibana-deployment.yaml [hunk 1]
```diff
@@ -2,6 +2,14 @@ apiVersion: extensions/v1beta1
...
 metadata:
   name: {{.Values.name}}
+  labels:
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/component: infrastructure
+    app.kubernetes.io/part-of: logging-central
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```
Comment (mikeln) [inline charts/kibana/templates/kibana-deployment.yaml]: should use a _helper  template.  A namevesion tempalte needs to added (like in the other charts) and used here.

File: charts/kibana/templates/kibana-ingress.yaml [hunk 2]
```diff
@@ -9,6 +9,14 @@ metadata:
...
     {{- end }}
   {{- end }}
+  labels:
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/component: infrastructure
+    app.kubernetes.io/part-of: logging-central
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```
Comment (mikeln) [inline charts/kibana/templates/kibana-ingress.yaml]: see above

File: charts/kibana/templates/kibana-secret.yaml [hunk 3]
```diff
@@ -3,6 +3,14 @@ apiVersion: v1
...
 metadata:
   name: {{ .Values.name }}-kibana-auth
+  labels:
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/component: infrastructure
+    app.kubernetes.io/part-of: logging-central
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```
Comment (mikeln) [inline charts/kibana/templates/kibana-secret.yaml]: see above

File: charts/kibana/templates/kibana-service.yaml [hunk 4]
```diff
@@ -6,6 +6,13 @@ metadata:
     app: {{.Values.app}}
     kubernetes.io/cluster-service: "true"
     kubernetes.io/name: "Kibana"
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/component: infrastructure
+    app.kubernetes.io/part-of: logging-central
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```
Comment (mikeln) [inline charts/kibana/templates/kibana-service.yaml]: see above

File: charts/kibana/templates/secret.yaml [hunk 5]
```diff
@@ -7,6 +7,13 @@ metadata:
     app: {{ template "name" . }}
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/component: infrastructure
+    app.kubernetes.io/part-of: logging-central
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```
Comment (mikeln) [inline charts/kibana/templates/secret.yaml]: see above

**Comment by mikeln** [comment]
_helper needs to be fixed:

```
{{- define "namewithversion" -}}
{{- $name := default .Chart.Name .Values.nameOverride -}}
{{- printf "%s-%s" $name .Chart.Version | replace "+" "_" | trunc 63  -}}
{{- end -}}
```

File: charts/kibana/templates/_helpers.tpl [hunk 6]
```diff
@@ -14,3 +14,13 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
...
 {{- end -}}
+
+{{/*
+Create a name with the version appended. This is used to set the `chart`
+label.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "namewithversion" -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- printf "%s-%s" $name .Chart.Version | replace "+" "_" | trunc 63  -}}
```
Comment (mikeln) [inline charts/kibana/templates/_helpers.tpl]: extra space after 63 may cause an error...

**Comment by michaelamiethyst** [comment]
I've made the .versionfile and _helpers changes.  Thank you.

**Pull request merged**